ui-tests/src/test/java/org/openmrs/reference/LoginTest.java

```diff
@@ -33,46 +36,45 @@ public void verifyModulesAvailableOnHomePage() throws Exception {
         assertTrue(homePage.isDataManagementAppPresent());
         assertTrue(homePage.isConfigureMetadataAppPresent());
         assertTrue(homePage.isSystemAdministrationAppPresent());
-    }
-
+    } 
```

Comment:
do you see the green above here, it's a trailing space

Reply:
Thanks let me correct them.

Reply:
Resolved thanks